lib/takeaway.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'menu'
+require_relative 'keys'
+require 'twilio-ruby'
+
+class Takeaway
+  attr_reader :menu, :order, :order_total
+
+  def initialize(menu = Menu.new, order = {}, order_total = 0)
+    @menu = menu
+    @order = order
+    @order_total = order_total
+  end
+
+  def read_menu
+    @menu.dishes
+  end
+
+  def place_order(dish, quantity = 1)
+    @order[dish] = quantity
+    return "#{quantity}x #{dish}(s) added to your order"
+  end
+
+  def basket_summary
+    summary = ""
+    @order.each {
+      |dish, quantity| summary << "#{dish} x#{quantity} = £#{menu.dishes[dish] * quantity}, "
+    }
+    summary.chop.chop
+  end
+
+  def total
+    @order.each {
+      |dish, quantity| @order_total += menu.dishes[dish] * quantity
+    }
+    return @order_total
+  end
+
+  def send_message
```

Comment:
With the use of the keys class that you've created, if I cloned your repo to my computer, the program wouldn't work because I wouldn't have your keys class. If you use environment variables on the other hand, I could simply set my own environment variables and the program would work for me.